main.nf

```diff
@@ -120,6 +110,18 @@ process gather_fastqs {
         final_sample_type = 'single-end'
     }
     template(task.ext.template)
+    
+    stub:
+    """
+    mkdir fastqs
+    mkdir extra
+    touch /*-error.txt
+    touch /fastqs/${sample}*.fastq.gz
+    touch /extra/*.gz
+    touch ${task.process}/*
```

Comment:
This doesn't seem right - this will created a file with the name of the task.

Reply:
@abhi18av when i use touch, should i use the  * ? I'm not sure if  this asterisk have a function

Reply:
Yup, that's why you need to test what you put into the `script` / `shell` / `stub` directives.